WolvenKit/Views/Editors/AutomaticNodifyEditor.cs

```diff
@@ -30,14 +30,17 @@ protected override void OnMouseMove(MouseEventArgs e)
 
             if (CurrentMousePosition != PreviousMousePosition)
             {
-                MouseLocation = TransformPosition(CurrentMousePosition);
 
+                //MouseLocation = TransformPosition(CurrentMousePosition);
+                // ^ `TransformPosition` was removed from NodifyEditor. Rather than trying to figure out the "correct" way to fix it,
+                // I hard coded it to what the definition of the function _was_ just to get things to build correctly on Nodify 3.0.0
+                MouseLocation = new Point((ViewportLocation.X + CurrentMousePosition.X) / ViewportZoom, (ViewportLocation.Y + CurrentMousePosition.Y) / ViewportZoom);
                 if (IsMouseCaptured)
                 {
                     // Panning
                     if (e.MiddleButton == MouseButtonState.Pressed)
                     {
-                        Offset -= CurrentMousePosition - PreviousMousePosition;
+                        ViewportLocation -= CurrentMousePosition - PreviousMousePosition;
```

Comment:
`CurrentMousePosition` and `PreviousMousePosition` cannot be relative to the `ItemsHost` because that control changes its transform based on the mouse delta, hence we need to scale it by `ViewportZoom`.

`ViewportLocation -= (CurrentMousePosition - PreviousMousePosition) / ViewportZoom;`

> I am thinking of simplifying this in the future.

Reply:
Ah, that makes sense. When testing it to make sure it worked, i noticed that zoomed out was annoyingly slow to pan.